clusters/resource_cluster.go

```diff
@@ -199,7 +199,14 @@ func resourceClusterRead(ctx context.Context, d *schema.ResourceData, c *common.
 		return err
 	}
 	d.Set("url", c.FormatURL("#setting/clusters/", d.Id(), "/configuration"))
-	if d.Get("library.#").(int) == 0 {
+	var isExporter bool
+	p, ok := ctx.Value(common.Provider).(*schema.Provider)
+	if !ok {
+		isExporter = false
+	} else {
+		isExporter = p.TerraformVersion == "exporter"
+	}
```

Comment:
Suggestion: let's make this a function in the `common` package so that we can reuse it in the future.

Reply:
Done...